docs/server/elasticsearch-setup.md

```diff
@@ -7,11 +7,8 @@ sidebar_label: Set up Elasticsearch
 ## Overview
 
 You can integrate [Elasticsearch](https://www.elastic.co/elasticsearch/) with the Temporal Server to enhance [Workflow search](/docs/server/workflow-search) functionality.
-The [Server versions and dependencies](/docs/server/versions-and-dependencies/#workflow-search) page describes which versions require the use of Kafka, and which ones do not.
```

Comment:
Instead of removing, maybe just update to:
The [Server versions and dependencies] page lists supported versions of Elasticsearch.